### PR TITLE
Expand whitelist, specially with default groovy methods

### PR DESCRIPTION
**Pull request by ydubreuil**
@reviewbybees

### CODE REVIEW

**Comment by rsandell** [comment]
:bee:

**Comment by ydubreuil** [comment]
@reviewbybees done

File: src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist [hunk 1]
```diff
@@ -7,6 +7,7 @@ method java.lang.CharSequence length
 method java.lang.Comparable compareTo java.lang.Object
 method java.lang.Object getClass
 method java.lang.Object equals java.lang.Object
+method java.lang.Object getClass
```
Comment (jglick) [inline src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist]: And what do you expect to do with the `Class` that would be safe? Generally anything smelling of reflection should be avoided unless it has been analyzed carefully.

Comment (jglick) [inline src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist]: …and anyway this entry was already there on line 8.

Comment (ydubreuil) [inline src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist]: @jglick I tried to be nasty with `Class` but failed to do anything bad. As you can see in the following stack trace, Sandbox is preventing use of reflexion:

```
org.jenkinsci.plugins.scriptsecurity.sandbox.RejectedAccessException: Scripts not permitted to use method java.lang.Class getField java.lang.String
	at org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.StaticWhitelist.rejectMethod(StaticWhitelist.java:150)
```

I will remove the duplicated line

**Comment by jglick** [comment]
:bee:

**Pull request merged**